package-lock.json

```diff
@@ -821,7 +821,8 @@
     "@types/chai": {
       "version": "4.2.14",
       "resolved": "https://registry.npmjs.org/@types/chai/-/chai-4.2.14.tgz",
-      "integrity": "sha512-G+ITQPXkwTrslfG5L/BksmbLUA0M1iybEsmCWPqzSxsRRhJZimBKJkoMi8fr/CPygPTj4zO5pJH7I2/cm9M7SQ=="
+      "integrity": "sha512-G+ITQPXkwTrslfG5L/BksmbLUA0M1iybEsmCWPqzSxsRRhJZimBKJkoMi8fr/CPygPTj4zO5pJH7I2/cm9M7SQ==",
+      "dev": true
```

Comment:
I see package-lock.json changes but no updates to package.json. Is this file accidentally included?

Reply:
This is from running npm install after deleting my node modules file. I didn't make any other changes